addons/mail/static/src/components/attachment/attachment_tests.js

```diff
@@ -538,6 +539,49 @@ QUnit.test('auto layout with image', async function (assert) {
     );
 });
 
+QUnit.test('view attachment', async function (assert) {
+    assert.expect(3);
+
+    await this.start({
+        hasDialog: true,
+        async mockRPC(route, args) {
+            if (route.includes('web/image/750')) {
+                return;
+            }
```

Comment:
Strange to require this, web/image should be auto-excluded in mockserver probably.

Reply:
Also I thought there was already magic so that img src attributes where ignored in tests?

Reply:
> Also I thought there was already magic so that img src attributes where ignored in tests?

Not ignore, just redirected as mocked RPCs

Reply:
task-2282438